Add in link and download to Mural file

clare-brown opened the pull request:
Interaction Designer, Clare Brown (clare.brown@digital.hmrc.gov.uk) has created an editable Mural template of GOV.UK Design System Flow Diagram for all designers to use.

This will help designers map out their service and/or user journeys at a high-level. The template is based off designs that were created in Miro and hosted on GitHub which can be found here: https://github.com/paulmsmith/govuk-designsystem-flow-diagram-miro.

File needs to be hosted for download, uploaded and have instructions on how people can use the file.

christopherthomasdesign commented:
Hi Clare – thanks for sharing this.

Is there anywhere I can access the Mural template to review it? 

I'm not familiar with how Mural templates work. Do you have to download a file, or can the template be hosted online by Mural? So far we've generally linked to online resources on this page, rather than sharing a downloadable file.

clare-brown commented:
Hello Chris,

Thanks so much for the speedy reply :) You can access and review the Mural
template here:
https://app.mural.co/t/digitalhmrc9032/m/digitalhmrc9032/1658237396824/6a3459a0afd115f18f7cd3a698405acae62bbf93?sender=u88f21b544bf2ceb924a42629

As for the template, it will be hosted online by Mural via a HMRC Mural
board I have set up (this is the one I linked you to) it has instructions
on how to copy over the components to your own Mural board so you can get
started using it. I hope this is all ok?

It will be really exciting to see my link on the Resources and tools page
under the 'Create flow diagrams' section. A good title would be 'GOV.UK
Design System Flow Diagrams for Mural - A Mural file for creating flow
diagrams of GOV.UK services.'

Thanks,

Clare Brown

On Mon, 8 Aug 2022 at 11:57, Chris Ballantine-Thomas <
***@***.***> wrote:

> Hi Clare – thanks for sharing this.
>
> Is there anywhere I can access the Mural template to review it?
>
> I'm not familiar with how Mural templates work. Do you have to download a
> file, or can the template be hosted online by Mural? So far we've generally
> linked to online resources on this page, rather than sharing a downloadable
> file.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alphagov/govuk-design-system/pull/2281#issuecomment-1207972287>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZQTHMC37NG7Q5HU7Y436MTVYDRZRANCNFSM54Z6USRA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Clare Brown

Interaction Designer, Economic Crime Levy

Email: ***@***.*** ***@***.***>*

christopherthomasdesign commented:
Thanks for sharing @clare-brown ! I've just reviewed the file against our [contribution criteria for resources](https://design-system.service.gov.uk/community/contribution-criteria/#developing-a-community-resource-or-tool), and I think it's looking good:

☑️ it has a clear owner
☑️ it cannot be mistaken for an official part of the Design System
☑️ it has documentation explaining how to use it, contribute to it and get support for it
**N/A** support tickets are answered within 1 month
**N/A** it’s clearly documented which version of GOV.UK Frontend it supports
**N/A** it’s no more than 3 months behind the latest version of GOV.UK Frontend
☑️ design resources are consistent with the styles, components and patterns in the GOV.UK Design System
**N/A** development resources output the same markup as GOV.UK Frontend
☑️ it is not being charged for or used to promote a private business
**N/A** it’s compatible with the [Service Standard](https://www.gov.uk/service-manual/service-standard) and [Technology code of practice](https://www.gov.uk/government/publications/technology-code-of-practice/technology-code-of-practice)

Would you be able to update the PR to use the Mural link you shared, and to put the link in the flow diagram section? I can do it if you're unable to.

Thanks again!

clare-brown commented:
Hello @christopherthomasdesignthanks for reviewing my file against the contribution criteria. Glad to hear it's good to go.

I've updated the PR with the Mural link and added the link to the flow diagram section of the [Resources and tools page](https://design-system.service.gov.uk/community/resources-and-tools/). Hopefully, this is all ok? Looking forward to sharing the link with my design community! 

Thanks,

Clare Brown

christopherthomasdesign commented on src/community/resources-and-tools/index.md.njk:
```diff
@@ -34,6 +34,9 @@ SVG images for creating flow diagrams of GOV.UK services.
 
 ## Create prototypes and wireframes
 
+[MURAL flow diagram](https://github.com/link-tbc) -
```
The link still isn't in here...

Then a couple of small details, sorry for being pernickety!

1. I'm not sure whether they prefer to stylise it as MURAL or Mural, but either way we should be consistent here
2. We order the resources alphabetically, so it should move to under the Figma link

Happy to fix all this myself if easier for you.

clare-brown commented:
Hello @christopherthomasdesign

Thanks for spotting that. I've changed it to MURAL in my design files and
on my GitHub public link.

If you could add the link in for me, that would be grand, thanks. It's:
https://github.com/clare-brown/govuk-designsystem-flow-diagram-mural

Also, if it were to sit under the Figma link, it would be in the wrong
section under 'Create prototypes and wireframes' but my resource is a flow
diagram, so I thought it would be better placed under the 'Create flow
diagrams' section.

Hope this is all good and thank you for all your help :)


On Thu, 18 Aug 2022 at 13:49, Chris Ballantine-Thomas <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In src/community/resources-and-tools/index.md.njk
> <https://github.com/alphagov/govuk-design-system/pull/2281#discussion_r949091414>
> :
>
> > @@ -34,6 +34,9 @@ SVG images for creating flow diagrams of GOV.UK services.
>
>  ## Create prototypes and wireframes
>
> +[MURAL flow diagram](https://github.com/link-tbc) -
>
> The link still isn't in here...
>
> Then a couple of small details, sorry for being pernickety!
>
>    1. I'm not sure whether they prefer to stylise it as MURAL or Mural,
>    but either way we should be consistent here
>    2. We order the resources alphabetically, so it should move to under
>    the Figma link
>
> Happy to fix all this myself if easier for you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alphagov/govuk-design-system/pull/2281#pullrequestreview-1077286798>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZQTHMCPHFH7MP2YXJKAF6LVZYWMXANCNFSM54Z6USRA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Clare Brown

Interaction Designer, Economic Crime Levy

Email: ***@***.*** ***@***.***>*

36degrees commented:
We've got in a bit of a muddle with multiple PRs for this change, but I can confirm that this change has now been made in PR #2318.

Thanks for your contribution, @clare-brown! I'm going to close this PR now as it's no longer needed.

clare-brown commented:
Ps. sorry, Chris the link to the Mural was incorrect in my last email. This
is the correct link:
https://app.mural.co/t/digitalhmrc9032/m/digitalhmrc9032/1660126757244/a18e7737b3a8c9b507dd02e5cafcb73e660ddaf5?sender=u88f21b544bf2ceb924a42629

Thanks :)

On Wed, 10 Aug 2022 at 11:16, Clare Brown ***@***.***>
wrote:

> Hello Chris,
>
> Thanks so much for the speedy reply :) You can access and review the Mural
> template here:
>
> https://app.mural.co/t/digitalhmrc9032/m/digitalhmrc9032/1658237396824/6a3459a0afd115f18f7cd3a698405acae62bbf93?sender=u88f21b544bf2ceb924a42629
>
> As for the template, it will be hosted online by Mural via a HMRC Mural
> board I have set up (this is the one I linked you to) it has instructions
> on how to copy over the components to your own Mural board so you can get
> started using it. I hope this is all ok?
>
> It will be really exciting to see my link on the Resources and tools page
> under the 'Create flow diagrams' section. A good title would be 'GOV.UK
> Design System Flow Diagrams for Mural - A Mural file for creating flow
> diagrams of GOV.UK services.'
>
> Thanks,
>
> Clare Brown
>
> On Mon, 8 Aug 2022 at 11:57, Chris Ballantine-Thomas <
> ***@***.***> wrote:
>
>> Hi Clare – thanks for sharing this.
>>
>> Is there anywhere I can access the Mural template to review it?
>>
>> I'm not familiar with how Mural templates work. Do you have to download a
>> file, or can the template be hosted online by Mural? So far we've generally
>> linked to online resources on this page, rather than sharing a downloadable
>> file.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/alphagov/govuk-design-system/pull/2281#issuecomment-1207972287>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AZQTHMC37NG7Q5HU7Y436MTVYDRZRANCNFSM54Z6USRA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
> Clare Brown
>
> Interaction Designer, Economic Crime Levy
>
> Email: ***@***.*** ***@***.***>*
>
>

-- 
Clare Brown

Interaction Designer, Economic Crime Levy

Email: ***@***.*** ***@***.***>*
